### DISCUSSION TITLE
Documentation completely lacking on major version increment

### DISCUSSION THREAD

**Discussion by animaxcg** [semantic-release/semantic-release#2576] [Commit Convention]
### Current behavior

Note the newline:
```
git commit -m"perf(MY-ISSUE): asdf
BREAKING CHANGE: asdf"
```

### Expected behavior

the former behavior with no newlines in commit message was nice and much better but if there is no desire to go back to it this should at very least be documented much better... The only words about it are: "BREAKING CHANGE:" must be included in the footer.. git has no concept of a footer.. this is the only place in the world I have heard 'footer" used for commit message.. "newline in commit message" should be used. very unclear and not consistent with fix/feat defaults.

```
git commit -m"perf(MY-ISSUE): asdf BREAKING CHANGE: asdf" 
#or
git commit -m"perf(MY-ISSUE): major increment
```

### `semantic-release` version

17+

### CI environment

N/A

### `semantic-release` configuration

{
  "branches": ["main", "next"],
  "plugins": [
    "@semantic-release/commit-analyzer",
    "@semantic-release/release-notes-generator",
    "@semantic-release/git"
  ]
}

### CI logs

N/A easy to test

**Comment by travi** [comment]
semantic-release enables using a commit convention to automate steps of a release workflow. originally semantic-release enabled the angular convention, which later evolved into the definition of the conventional-commits specification (note: semantic-release still defaults to the use of the angular convention at this point). the terminology used in our docs is based on using those conventions, which can be found [here](https://www.conventionalcommits.org/en/v1.0.0/), for example. 

> the former behavior with no newlines in commit message was nice and much better but if there is no desire to go back to it

semantic-release/the angular convention has never enabled the use of the `BREAKING CHANGE: ` token in the commit description, so this is not a regression from previous behavior and is also not behavior that would be changed in semantic-release outside of it changing in the external specification. 

if you want behavior that does not align with the provided convention options, you can define your own. 

> this should at very least be documented much better

we are always open to improvements to our documentation, so please feel free to submit a PR that makes it more clear where the footer term comes from and we could discuss further from there.
